themes/kiali/layouts/_default/baseof.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+<head>
+  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+  <link rel="icon" type="image/png" href="{{ "img/favicon.png" | relURL }}"/>
+  <title>{{ block "title" . }}{{ .Site.Title }}: Service mesh observability and configuration{{ end }}</title>
+
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.9.0/css/fontawesome.min.css"
+    integrity="sha256-AaQqnjfGDRZd/lUp0Dvy7URGOyRsh8g9JdWUkyYxNfI=" crossorigin="anonymous" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.9.0/css/brands.min.css" 
+    integrity="sha256-u8123o+sLy8uk0Du9H0Ub+KinAoHanzGsBqDkWHY1f8=" crossorigin="anonymous" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.9.0/css/solid.min.css"
+    integrity="sha256-3FfMfpeajSEpxWZTFowWZPTv7k3GEu7w4rQv49EWsEY=" crossorigin="anonymous" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+  <link rel="stylesheet" href="{{ "css/kiali.css" | relURL }}" />
+
+  <!--
+    Syntax hightlighting: https://highlightjs.org/download/
+    Languages pack: CSS, HTTP, JavaScript, Bash, Makefile, Diff, JSON, HTML/XML, Java, Properties,
+      Shell Session, AsciiDoc, Dockerfile, Typescript, YAML, plaintext.
+  -->
+  <link rel="stylesheet" href="{{ "css/highlight.css" | relURL }}">
+  <script src="{{ "js/highlight.pack.js" | relURL }}"></script>
```

Comment:
I think the script tag should have a type element, that describes the media type - especially when we tighten the content security policy. See e.g. https://github.com/kiali/kiali.io/pull/146/files#diff-8ab16b2cae0b26a574b75e8e5c19e378

Reply:
Fixed.